contributors/devel/help-wanted.md

```diff
@@ -1,13 +1,108 @@
+We use two labels `help wanted` and `good first issue` to identify issues that have been
+specially groomed for new contributors. The `good first issue` label is a
+subset of `help wanted` label, indicating that members have committed to
+providing extra assistance for new contributors. All `good first issue` items also have the `help wanted` label.
+
 # Help Wanted
 
 Items marked with the `help wanted` label need to ensure that they are:
 
-- Sufficiently actionable: clear description of what should be done
-- Tractable for new/casual contributors: there is documentation how that type of change should be made
-- Goldilocks priority: Not too high that a core contributor should do it, but not too low that it isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
-- Up to date: Often these issues become obsolete and have already been done, are no longer desirable, no longer make sense, change in priority, change in difficulty, etc.
+- **Low Barrier to Entry**
+  It should be tractable for new contributors. Documentation on how that type of change should be made should already exist.
```

Comment:
I think this should be in a newline? I think that's what you had done initially with the `</br`>. It looked better that way! :)

Without using the `</br>`, a newline between line 10 and 11 should achieve the same result.